### ISSUE TITLE
Caffe training prototxt for transfer-learning from the Kinectics pretrain models

### ISSUE THREAD

**Issue by ks-mengxi** [yjxiong/temporal-segment-networks#136]
Hi, yjxiong,

I realized that your group has released the pretrained models of Kinectics.
But I couldn't find the caffe training prototxts for transfer learning from the pretrained models.
Are you going to release these prototxts?
Thank you very much.

**Comment by Tonyfy** [comment]
@ks-mengxi 
Hi ,I can't download the last two models , ``` V3 RGB PRETRAINED WEIGHTS``` and ```V3 FLOW PRETRAINED WEIGHTS```.
Dose anybody meet the same question?

**Comment by ks-mengxi** [comment]
@Tonyfy 
Yes. I could download it quite smoothly several hours ago.
Not sure now.

**Comment by Tonyfy** [comment]
@ks-mengxi  I got them now!

**Comment by Tonyfy** [comment]
@ks-mengxi and according to the deploy.prototxt in the zip file, we can easily got train_val.prototxt, just modify the data layer and add loss layer.

**Comment by ks-mengxi** [comment]
@Tonyfy I see. I will try then.

**Comment by Tonyfy** [comment]
@ks-mengxi I think the biggest obstacle on training is the dataset, I am crawling the kinetics dataset which may cost too much much much time...

**Comment by yjxiong** [comment]
For transfer learning UCF101, see #132 .
For temporal action detection with SSN, Kinetics pre-trained models are already integrated into the released toolbox.
